tests/sles4sap/crash/azure/crash_azure.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary:  Executes a crash scenario on azure.
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use mmapi 'get_current_job_id';
+use serial_terminal 'select_serial_terminal';
+use version_utils 'is_sle';
+use sles4sap::azure_cli;
+use publiccloud::instance;
+use utils;
+
+use constant DEPLOY_PREFIX => 'clne';
+
+sub run {
+    my ($self) = @_;
+
+    # Crash test
+    my $ssh_cmd = get_required_var('SSH_CMD');
+    my $vm_ip = get_required_var('VM_IP');
+    my $instance = publiccloud::instance->new(public_ip => $vm_ip);
```

Comment:
Why do we need the command in `SSH_CMD` if we're later creating a `publiccloud::instance` object which should be able to generate that command for us? See: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/instance.pm#L57